samples/IISSample/project.json

```diff
@@ -15,7 +14,12 @@
   "frameworks": {
     "net451": { },
     "netstandardapp1.5": {
-      "imports": "portable-net451+win8"
+      "imports": [
+        "dnxcore50", "portable-net451+win8"
+      ],
+      "dependencies": {
+        "NETStandard.Library": "1.5.0-*"
```

Comment:
The feed use (aspnetcidev/aspnetvolatile) only has compatible packages (i.e. we won't take rc3 packages)